client/state/happiness-engineers/test/reducer.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import {
+	HAPPINESS_ENGINEERS_FETCH,
+	HAPPINESS_ENGINEERS_RECEIVE,
+	HAPPINESS_ENGINEERS_FETCH_FAILURE,
+	HAPPINESS_ENGINEERS_FETCH_SUCCESS,
+	SERIALIZE,
+	DESERIALIZE
+} from 'state/action-types';
+import { useSandbox } from 'test/helpers/use-sinon';
+import reducer, { requesting, items } from '../reducer';
+
+describe( 'reducer', () => {
+	useSandbox( ( sandbox ) => sandbox.stub( console, 'warn' ) );
+
+	it( 'should include expected keys in return value', () => {
+		expect( reducer( undefined, {} ) ).to.have.keys( [
+			'requesting',
+			'items'
+		] );
+	} );
+
+	describe( 'requesting()', () => {
+		it( 'should default to an empty object', () => {
```

Comment:
Actually, I think it should default to `false`, because we can assume that initially, we won't be requesting anything.